src/main/java/Duke.java

```diff
@@ -1,10 +1,120 @@
+
+    public static String filePath = Paths.get(System.getProperty("user.dir"), "data/duke.txt").normalize().toString();
+
+
+    public static void echo(String line) {
+        System.out.println(LINE + LINE_SEPARATOR + INDENT + line + LINE_SEPARATOR + LINE);
+    }
+
+    public static void loadData() {
+        try {
+            DataSaver.checkFileExist("data/duke.txt");
+            DataSaver.loadFileContents("data/duke.txt");
+        } catch (FileNotFoundException e) {
+            System.out.println(e.getMessage());
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    public static void saveData() {
+        try {
+            Path filePath = Paths.get("data/duke.txt");
+            Files.createDirectories(filePath.getParent());
+            DataSaver.writeFileContents("data/duke.txt");
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    public static void executeRequest() {
+        String line;
+        Scanner in = new Scanner(System.in);
+        int taskCount = 0;
+        Task[] tasks = new Task[100];
+
+        line = in.nextLine();
+        String[] words = line.split(" ");
+
+        while (!line.equals("bye")) {
+            System.out.println(LINE);
+            try {
+                switch (words[0]) {
+                case "list":
+                    TaskHandler.printTaskList();
+                    break;
+                case "done":
+                    TaskHandler.markTaskAsDone(words);
+                    saveData();
+                    break;
+                case "deadline":
+                    TaskHandler.addDeadline(line);
+                    saveData();
+                    break;
+                case "event":
+                    TaskHandler.addEvent(line);
+                    saveData();
+                    break;
+                case "todo":
+                    TaskHandler.addTodo(line);
+                    saveData();
+                    break;
+                case "help":
+                    TaskHandler.showHelp();
+                    break;
+                case "delete":
+                    TaskHandler.deleteTask(words);
+                    saveData();
+                    break;
+                default:
+                    TaskHandler.handleWrongCommand();
+                    break;
+                }
+            } catch (DukeException e) {
+                System.out.println(e.getMessage());
+            } catch (NumberFormatException e) {
+                System.out.println(e.getMessage());
+            }
+            System.out.println(LINE);
+            line = in.nextLine();
+            words = line.split(" ");
+        }
+    }
+
+    public static void greetUser() {
+        echo("Hello! I'm Duke." + LINE_SEPARATOR_AND_INDENT + "What can I do for you?" +
+                LINE_SEPARATOR + LINE_SEPARATOR_AND_INDENT + HELP_INSTRUCTIONS);
+    }
```

Comment:
Any particular reason for having an `echo` that simply calls `System.out.println()`? This may seem overdoing SLAP.
Instead, consider renaming the `greetUser` to `printWelcomeMessage` or something similar that conveys the message clearly to anyone who is reading your code.